source/hardware/controllers/denon_mc7000.rst

```diff
@@ -49,12 +49,14 @@ improve the mapping, then please discuss it in the `Denon MC7000
 Mapping <https://mixxx.discourse.group/t/denon-mc7000-mapping/18235>`__
 thread.
 
+.. _denon_mc7000_userVariables:
```

Comment:
Please use dashes instead of underscore and make anchors all lowercase. That's what already have elsewhere.

```suggestion
.. _denon-mc7000-uservariables:
```

Reply:
all variables changed to lowercase